skills/skill-agent-topology/SKILL.md

```
@@ -0,0 +1,177 @@
+---
+name: skill-agent-topology
+description: "Audit whether a multi-agent setup earns its coordination cost — use before adding an agent, or when a workflow feels slow or agents agree without adding signal"
+---
+
+> **Host: Codex CLI** — This skill was designed for Claude Code and adapted for Codex.
```

markdownlint-cli2 (0.23.1)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)